picard/util/__init__.py

```diff
@@ -459,4 +458,20 @@ def convert_to_string(obj):
         return str(obj)
 
 
+def htmlescape(string):
+    return cgi.escape(string)
+
+
+def json_load(data):
```

Comment:
Other functions in this file (`build_qurl`, `htmlescape`, ...) are named `verb_something`, not the other way around.